src/languages/en.js

```diff
@@ -174,11 +174,13 @@ export default {
         deleteConfirmation: 'Are you sure you want to delete this comment?',
     },
     reportActionsView: {
+        beginningOfChatHistoryAdminRoomPartOne: ({workspaceName}) => `Collaboration among ${workspaceName} admins starts here! 🎉\nUse `,
+        beginningOfChatHistoryAdminRoomPartTwo: ' to chat about topics such as workspace configurations and more.',
+        beginningOfChatHistoryAnnounceRoomPartOne: ({workspaceName}) => `Collaboration between all ${workspaceName} members starts here! 🎉\nUse `,
+        beginningOfChatHistoryAnnounceRoomPartTwo: ({workspaceName}) => ` to chat about anything ${workspaceName} related.`,
+        beginningOfChatHistoryUserRoomPartOne: 'Collaboration starts here! 🎉\nUse this space to chat about anything ',
+        beginningOfChatHistoryUserRoomPartTwo: ' related.',
```

Comment:
Why is this on a separate line?

Reply:
@stitesExpensify Refer below this how the above translations are used thats why separated,
```js
        <Text style={styles.textAlignCenter}>
            {roomWelcomeMessage.phrase1} // allPartOne
        </Text>
        <Text style={[styles.textStrong]}>
            {props.report.reportName}. // #roomName
        </Text>
        <Text>
            {roomWelcomeMessage.phrase2} // partTwo
        </Text>
```

Reply:
> I'm a little confused why we do \n
I thought that was a line break at end of the sentence. Do you want to remove it?

Reply:
Ah gotcha, thanks!

Reply:
No, that makes sense. I misread the code originally and thought that the separate `<text>` was putting it on a different line